docs/diagrams/AddTaskSequenceDiagram.puml

```diff
@@ -33,7 +31,7 @@ AddressBookParser -> AddCommandParser : parse("-e...")
 activate AddCommandParser
 
 activate TaskCommandParser
-AddCommandParser -> TaskCommandParser : parse("-e...")
+AddCommandParser -> TaskCommandParser : parseEvent("-e...")
```

Comment:
For Add Tasks do we not need a ParserUtil class?

Reply:
Ok I will add it in the next pr, thanks for mentioning it